docs/logs/correlation/README.md

```diff
@@ -3,7 +3,8 @@
 Starting from `Microsoft.Extensions.Logging` version `5.0`, logs can be
 correlated with distributed tracing by enriching each log entry with the
 information from the enclosing `Activity`. This can be achieved by enabling the
-`ActivityTrackingOptions`:
+`ActivityTrackingOptions`. In a non-host console app, it can be achieved as
```

Comment:
Probably no need to change this line as it is already covered later? (adding this line might actually cause confusion as what is shown right below is the console scenario).

Reply:
nit: if this change is necessary, link "non-host console app" to https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-3.1#non-host-console-app